wp-content/themes/goonj-crm/assets/styles/goonj-main.scss

```
@@ -680,23 +680,26 @@
     }
   }
 }
-// Inline validation error on the street address field, on phones.
+// Inline validation errors on the 80(G) fields — street address and PAN — on
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)